public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -11,17 +11,33 @@ const TopSection: FC<any> = () => {
     text: 'Search',
     icon: 'search',
   };
-
+  const dbaasLink = {
+    id: 'dbaas',
+    icon: 'database',
+    text: 'DBaaS',
+    url: '/graph/d/pmm-dbaas/pmm-dbaas',
+  };
   const onOpenSearch = () => {
     getLocationSrv().update({ query: { search: 'open' }, partial: true });
   };
+  const [showDBaaS, setShowDBaaS] = useState(false);
+
+  // TODO: once DBaaS is enabled by default move it to config file (api/index.go)
+  useEffect(() => {
+    if (config.bootData.user.isSignedIn) {
+      getBackendSrv()
+        .post('http://localhost/v1/Settings/Get')
+        .then(({ settings }) => setShowDBaaS(settings.dbaas_enabled));
+    }
+  }, []);
 
   return (
     <div className="sidemenu__top">
       <TopSectionItem link={searchLink} onClick={onOpenSearch} />
       {mainLinks.map((link, index) => {
         return <TopSectionItem link={link} key={`${link.id}-${index}`} />;
       })}
```

Comment:
```suggestion
      {mainLinks.map((link, index) => (
        <TopSectionItem link={link} key={`${link.id}-${index}`} />;
      ))}
```

Reply:
This is Grafana code so won't change it to avoid conflicts when we pull from upstream to get the latest version, and because we don't want to maintain this file (since our code will be removed from here when DBaaS is always enabled)